crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -62,35 +106,19 @@ pub enum NodeConfigs<T> {
         configs: Vec<NodeConfigs<T>>,
         /// Run conditions applied to everything in the tuple.
         collective_conditions: Vec<BoxedCondition>,
+        /// Metadata for `NodeConfigs`
+        metadata: T::GroupMetadata,
         /// See [`Chain`] for usage.
         chained: Chain,
```

Comment:
```suggestion
```
This can be removed because it's provided by the `metadata: T::GroupMetadata` (recall `type GroupMetadata = Chain;`).